tuxemon/core/save_upgrader.py

```diff
@@ -0,0 +1,52 @@
+"""
+This module is for handling breaking changes to the save file.
+
+Renaming maps:
+  - Increment the value of SAVE_VERSION (e.g. from 1 to 2)
+  - Add an entry to MAP_RENAMES consisting of:
```

Comment:
Went into lots of detail here as map editors aren't expected to know python.

Reply:
good idea